eng/common/scripts/Package-Properties.ps1

```diff
@@ -76,81 +76,76 @@ function Get-PkgProperties
 {
     Param
     (
-        [Parameter(Mandatory = $true)]
         [string]$PackageName,
-        [Parameter(Mandatory = $true)]
+        [string]$ArtifactName,
```

Comment:
We could consider just having one parameter and call it "PackageOrArtifactName" and when we check we could first compare it with the PackageName and and then the ArtifactName. There shouldn't be any conflict because the should either be the same or unique depending on the language.